GVFS/GVFS.UnitTests.Windows/GVFS.UnitTests.Windows.csproj

```diff
@@ -194,6 +193,12 @@
       <Link>Readme.md</Link>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="..\GVFS.UnitTests\Service\MacServiceTests.cs" />

```

Comment:
It feels odd to have to forcefully exclude Mac-specific tests from the Windows compile. Did you consider having these tests live under GVFS.UnitTests.Mac.csproj?

Reply:
There is this single file which is Mac specific. Creating a separate `GVFS.UnitTests.Mac.csproj` project makes sense in the long run, as we add more Mac specific UT. 

Thoughts?